src/content/Concerts/02-03Fall.md

```diff
@@ -0,0 +1,40 @@
+---
+# Concert Name (e.g. "Fall Concert")
+concertName: "Fall Concert"
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#F58025"
+# Date YY-MM-DD (for sorting)
+date: "2002-10-00"
+# Season
+season: "2002/03"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: "bernstein"
+# Google Calendar link (if applicable)
+calendar: ""
+# Ticket link (if applicable)
+tickets: ""
+# Livestream link (if applicable)
+stream: ""
+# Youtube link (if applicable)
+youtube: ""
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = The School for Scandal
+Subtitle = Michael Pratt, Conductor
+
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Generaltime = October 2002
+
+Section
+
+Overline = Program
+Work = Samuel Barber ~ Overture to *The School for Scandal*, op. 5
+Work = Leonard Bernstein ~ *Fancy Free*, op. 45
```

Comment:
was it the full work or just the suite? I bet it's the suite (in whoich case make it match 18/19 season